docs/allocation.md

```diff
@@ -115,6 +115,8 @@ There are a number of advantages to using arraylets.
 
 The overall size of the Java heap is determined by two command-line options, `-Xms`, which sets the initial size of the heap, and `-Xmx`, which sets the maximum size of the heap. Finer tuning of the heap depends on the heap configuration that is being used by a GC policy. For example, an LOA within the heap can be sized by using the [`-Xloainitial`, `-Xloaminimum`, and `-Xloamaximum`](xloaminimum.md) command-line options. A nursery area within the heap can be sized by using the [`-Xmn`, `-Xmns`, and `-Xmnx`](xmn.md) command-line options. For more information about policies and the heap configurations that are used, see [GC policies](gc.md). To determine the values that are in use for the Java heap, use the `-verbose:sizes` option when you run your Java application.
 
+When the Java heap runs out of space, `OutOfMemoryError` exceptions are generated. If you are confident that your heap settings are appropriate for your application but are still receiving an `OutOfMemoryError` exception, check the Java dump file that gets automatically generated when the error occurs. A Java dump file can tell you more about what your application was attempting to do at the time of the error. For example, see the [Java OutofMemoryError](dump_javadump.md#java-outofmemoryerror) scenario.
+
```

Comment:
in 'see the [Java OutofMemoryError]', second 'O' should be upper case

Reply:
Thanks @amicic and @dmitripivkine for reviews/comments. Typo corrected.